phageAPI/populate.py

```diff
@@ -1,19 +1,79 @@
+#!/usr/bin/env python
+
 import os
-import fetch
-from IPython import embed
-from Bio import SeqIO
-from lxml import html, etree
+import sys
+import argparse
 import requests
 import pandas
 import pickle
-from prunedict import prunedict
+from lxml import html, etree
+from Bio import Entrez
+from Bio import SeqIO
 from tqdm import tqdm
 
+os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'phageAPI.settings')
+import django
+django.setup()
+
+from util.acc import read_accession_file
+from util.prunedict import prunedict
+from util import fetch
+from restapi.models import (
+    Organism,
+    Spacer,
+    Repeat,
+    OrganismSpacerRepeatPair,
+    AntiCRISPR
+)
+
+DATA_DIR = os.path.abspath(os.path.join(os.path.dirname(__file__), '../data'))
+
+
+def status(count, done=False):
```

Comment:
I don't think this is necessary; you've got a `tqdm` progress bar already. If you want more information there, then it should be added to the `tqdm` call. If there's no `tqdm` call, it can be added.